.github/actions/checkDeployBlockers/checkDeployBlockers.js

```diff
@@ -16,7 +16,7 @@ const run = function () {
             console.log('Checking for unverified PRs or unresolved deploy blockers', data);
 
             // Check the issue description to see if there are any unfinished/un-QAed items in the checklist.
-            const uncheckedBoxRegex = new RegExp(`-\\s\\[\\s]\\s(?:QA|${GithubUtils.ISSUE_OR_PULL_REQUEST_REGEX.source})`);
+            const uncheckedBoxRegex = /-\s\[\s]\s(?!Accessibility)/;
```

Comment:
Not necessary – since there's no opening bracket to go along with it JS won't confuse this with a character group. I can always change it if it gives you the heebie-jeebies 😄

Reply:
No harm in escaping it, so I'll just add that in.

Reply:
That's lame. I'd rather see `I'm explicitly looking for a literal closing bracket` rather than `No opening bracket so ¯\_(ツ)_/¯`

Reply:
Yeah, I actually agree with you but have a revulsion to squiggly lines in my IDE